src/clusterfuzz/_internal/bot/fuzzers/centipede/engine.py

```diff
@@ -114,8 +114,6 @@ def prepare(self, corpus_dir, target_path, build_dir):
     else:
       logs.log_warn('Unable to find sanitized target binary.')
 
-    arguments.append(f'--timeout_per_input={_TIMEOUT_PER_INPUT}')
```

Comment:
As discussed, let's bring this back, with the different timeouts for fuzzing vs reproduction.

Reply:
Yep, I will also change the `centipede-old` binary to the second most recent version to adapt to this change.